src/popup.tsx

```diff
@@ -27,12 +28,15 @@ export const Popup = () => {
         Calculate CO2 emissions
       </button>
       <div >
-        SCI: {calculateCarbon(selectedCountries)} gCO2eq
+        Total Data Received: {totalBytesReceived} bytes
       </div>
       <div >
-        Specific Carbon Emissions (gC02 per byte): {averageSpecificEmissions}
+        Specific Carbon Emissions (grams of C02 per byte): {Math.round(averageSpecificEmissions * 100) / 100}
       </div>
-      <SelectedCountries selectedCountries={selectedCountries} removeSelectedCountry={removeSelectedCountry} setCountryPercentage={setCountryPercentage}/>
+      <div >
+        Software Carbon Intensity: {Math.round(emissions * 100) / 100} grams of CO2
+      </div>
```

Comment:
Do we need both of these or can we just use one :D

Reply:
I quite like the idea of having emissions per byte, so that further down the line the client can understand how the location of their servers affects emissions; it might also be a good idea to also show the emissions per byte for each location

What do you think?